### ISSUE TITLE
feature request: Configurable color space

### ISSUE THREAD

**Issue by nowak-pl** [alvr-org/ALVR#2426]
Currently ALVR sets color space to DCI-P3 in client.
https://github.com/alvr-org/ALVR/blob/4d0d0ce59ac6ffe34b4e19f5fc8bfe25f6ab8202/alvr/client_openxr/src/lib.rs#L245

It would be nice to have selectable color space in streamer because content authored in Rec. 709 or sRGB doesn't look the best when expanded to P3 without proper conversion and then clipped to HMD LCD panel native color space.

**Comment by nowak-pl** [comment]
I'm only talking about sending to client which color space should it use in set_color_space call, no conversions on either end.

**Comment by zarik5** [comment]
Yeah it can be done. There is a openxr extension to do that. I'll accept PR to implement it.
